pyomo/contrib/gdpopt/tests/test_gdpopt.py

```diff
@@ -22,6 +22,7 @@
 from pyomo.common.collections import Bunch
 from pyomo.common.config import ConfigDict, ConfigValue
 from pyomo.common.fileutils import import_file, PYOMO_ROOT_DIR
+from pyomo.contrib.appsi.base import Solver
```

Comment:
Oh, good catch. Because of a long and stupid adventure trying to get the appsi test to skip properly when Gurobi is installed but has no license. But we actually don't need this import or the one below.